### DISCUSSION TITLE
Does this package support the Twitter Ads API?

### DISCUSSION THREAD

**Discussion by nbro10** [tweepy/tweepy#2154] [Questions]
There are at least 2 APIs provided by Twitter (or X). The Twitter API, which this package seems to provide a Python interface for, and the Twitter Ads API. My understanding is that `tweepy` only supports the Twitter API, so not the Twitter Ads API. Am I correct?

The reason why I am asking this is that the [`twitter-ads`](https://github.com/twitterdev/twitter-python-ads-sdk) package seems to be dead/unmaintained and now, after the changes they have been making to the APIs, it's also buggy.

**Comment by Harmon758** [comment]
You are correct; it does not.
The main reasons are because there was an official library and I've never had access to the Ads API.
